### PR TITLE
Make homepage loading timeout configurable for selenium

### PR DESCRIPTION
**Pull request by jhrcek**
@psiroky please review. If acceptable I'll create PR for 6.4.x as well
and then we'll need to add something like `-Dselenium.homepage.loading.timeout=30` to selenium jenkins jobs configuration.

### CODE REVIEW

File: kie-wb-smoke-tests/kie-wb-smoke-tests-main/src/main/java/org/kie/smoke/wb/selenium/model/HomePerspective.java [hunk 1]
```diff
@@ -1,56 +0,0 @@
...
-
-import org.kie.smoke.wb.selenium.util.PageObjectFactory;
-import org.kie.smoke.wb.selenium.util.Waits;
-import org.openqa.selenium.By;
-import org.openqa.selenium.NoSuchElementException;
-import org.openqa.selenium.WebDriver;
-import org.openqa.selenium.WebElement;
-import org.openqa.selenium.support.PageFactory;
-
-public class HomePerspective extends PageObject {
```
Comment (jhrcek) [inline kie-wb-smoke-tests/kie-wb-smoke-tests-main/src/main/java/org/kie/smoke/wb/selenium/model/HomePerspective.java]: This class was duplicated by mistake.

**Comment by psiroky** [comment]
There are three test failures. They seem to be related to these changes. @jhrcek could you please take a look?

**Comment by jhrcek** [comment]
Right. I didn't count with System.getProperty with default value declared to return empty String :-/
Should be fixed now.

**Comment by jhrcek** [comment]
The fails are caused by the timeout we're trying to overcome with this PR.

**Comment by jhrcek** [comment]
Dear Jenkins retest this, please

**Comment by psiroky** [comment]
Thanks @jhrcek, merged (with very minor adjustments, hope that's ok) by https://github.com/droolsjbpm/kie-wb-distributions/commit/b1475279b4891c8796bdc59f0f0b28577c692fad.

The tests are unfortunately still failing in the Jenkins (even with timeout 120s). I will need to debug it directly there.

**Comment by jhrcek** [comment]
@psiroky Sure I don't mind. I should have known better not to use magical constants. Thanks!

**Pull request closed without merging**